db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql

```diff
@@ -13,13 +13,112 @@
 -- Jun-2021		Ram Bhatt	   Added rows to transaction_type.csv
 -- Jun-2021		Ram Bhatt	   Added columns to registration list table
 -- Jun-2021 		Ram Bhatt	   Create crypto salt table.
--- July-2021		Ram Bhatt	   Added rows to transaction_type.csv
+-- Jul-2021		Ram Bhatt	   Added rows to transaction_type.csv
+-- Jul-2021		Ram Bhatt	   Multiple table changes on regprc db
 ----------------------------------------------------------------------------------------------------
 \c mosip_regprc sysadmin
 
+----------------------------------------------Multiple table level changes on regprc db-------------------------------------------------------
+
+ALTER TABLE regprc.individual_demographic_dedup DROP CONSTRAINT IF EXISTS fk_idemogd_reg CASCADE;
+ALTER TABLE regprc.reg_manual_verification DROP CONSTRAINT IF EXISTS fk_rmnlver_reg CASCADE;
+ALTER TABLE regprc.reg_bio_ref DROP CONSTRAINT IF EXISTS fk_regref_reg CASCADE;
+ALTER TABLE regprc.reg_lost_uin_det DROP CONSTRAINT IF EXISTS fk_rlostd_reg CASCADE;
+
```

Comment:
additionally -
ALTER TABLE regprc.registration_transaction DROP CONSTRAINT IF EXISTS fk_regtrn_reg CASCADE

then add workflow_instance_id as foreign key for registration_transaction table